doc/transport/input.rst

```diff
@@ -0,0 +1,240 @@
+
+→ View :source:`data/transport/technology.yaml` on GitHub
+
+Code lists for other MESSAGE sets (:file:`set.yaml`)
+----------------------------------------------------
+
+This file gives code lists for other MESSAGE concepts/sets/dimensions.
+
+→ View :source:`data/transport/set.yaml` on GitHub
+
+.. _transport-data-files:
+
+Input data files
+================
+
+:data:`.transport.files.FILES` gives a list of all data files.
+Through :func:`.transport.build.main` (ultimately, :func:`.transport.build.add_exogenous_data`), each of these files is connected to a :class:`genno.Computer` used for model-building, and its contents appear at the key given in the list below.
+
+.. admonition:: Example
+
+   Contents of the file :file:`freight-mode-share-ref.csv` are available at the key ``freight mode share:n-t:ref``.
+   The indicates the dimensionality of this quantity is :math:`(n, t)`.
+   The file has column headers "node", "technology", and "value".
+
+Not all files are currently or always used in model-building computations.
+Some submodules of :mod:`~message_data.model.transport` use additional data files via other mechanisms.
+Most of the files have a header comment including a precise description of the quantity, source of the data, and units of measurement; in some cases extended information is below (where a header comment would be too long).
+
+:file:`ldv-activity.csv` → ``ldv activity:n:exo``
+-------------------------------------------------
+
+Measure
+   Activity (driving distance) per light-duty vehicle
+Units
+   kilometre / year
+
+Notes
+~~~~~
+
+node = R12_AFR [1]_
+  Obtained from literature, based on estimates from South Africa. The reported value for South Africa is lower (18000 km/year, `source <https://blog.sbtjapan.com/car-info/what-mileage-is-good-for-a-used-car#:~:text=Average%20Mileage%20in%20South%20Africa,is%20just%20a%20general%20guideline>`__) than the one for Kenya (22000 km/year, `source <https://www.changing-transport.org/wp-content/uploads/2019_Updated-transport-data-in-Kenya.pdf>`__).
+
+node = R12_FSU [1]_
+  Based on Russia estimates (`source <https://eng.autostat.ru/news/17616/>`__).
+
+node = R12_NAM [1]_
+  Based on US estimates (`source <https://afdc.energy.gov/data/10309>`__`), Canada estimates tend to [be] lower in general.
+
+node = R12_PAO [1]_
+  Estimates for AU is 11000 in 2020, it's a sharp decrease from 12600 in 2018 (maybe a Covid effect?).
+  Whereas JP is 8532 (`source <https://www.mlit.go.jp/road/road_e/statistics.html>`__) in 2016.
+
+node = R12_PAS [1]_
+  Based on Singapore by `Chong et al. (2018) <https://doi.org/10.1016/j.enconman.2017.12.083>`__.
+
+node = R12_SAS [1]_
+  Based on India, mainly Delhi estimate by `Goel et al. (2015) <https://doi.org/10.1016/j.tbs.2014.10.001>`__.
+
+.. [1] A. Javaid, `message_data#180 (comment) <https://github.com/iiasa/message_data/issues/180#issuecomment-1944227441>`__.
```

Comment:
Are you going to migrate transport-related issues and PRs from message_data to have all in one place?

Reply:
TBD. I have mentioned this to @r-aneeque; I think we will likely migrate *pending* issues to message-ix-models, but leave completed issues and PRs where they are. We don't have any open PRs at this point.